drivers/media/i2c/irs1125.c

```diff
@@ -748,33 +759,22 @@ static const struct v4l2_ctrl_config irs1125_custom_ctrls[] = {
 		.id = IRS1125_CID_MOD_PLL,
 		.name = "Reconfigure modulation PLLs",
 		.type = V4L2_CTRL_TYPE_U16,
-		.flags = V4L2_CTRL_FLAG_HAS_PAYLOAD,
+		.flags = V4L2_CTRL_FLAG_HAS_PAYLOAD |
+			V4L2_CTRL_FLAG_EXECUTE_ON_WRITE,
```

Comment:
Do you really need to write it every time, even if it hasn't changed? I'm willing to believe it, but it's slightly unusual.

Reply:
No this is not needed, you are right.